### PR TITLE
RAIL-ify /web/perf... fixes #3519

### LINKED ISSUE #3519: Add RAIL to /web/fundamentals/performance
It is crazy that we don't mention RAIL in the performance section and it's buried away in the /web/tools (I understand the context).

It would be good to get the RAIL guidance front and center in /web/fundamentals/performance

### PR DESCRIPTION
**Pull request by kaycebasques**
@PaulKinlan LGTY?

I also changed the title to make it moar action-y

### CODE REVIEW

File: src/content/en/tools/_redirects.yaml [hunk 1]
```diff
@@ -1,10 +1,13 @@
 redirects:
 
+- from: /web/tools/chrome-devtools/evaluate-performance/rail
+  to: /web/fundamentals/performance/rail
+
```
Comment (addyosmani) [inline src/content/en/tools/_redirects.yaml]: /web/fundamentals/performance/rail 👍

File: src/content/en/fundamentals/performance/rail.md [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
 
 {# wf_updated_on: 2015-06-07 #}
 {# wf_published_on: 2015-06-07 #}
 
-# The RAIL Performance Model {: .page-title }
+# Measure Performance with the RAIL Model {: .page-title }
 
 {% include "web/_shared/contributors/megginkearney.html" %}
 
 RAIL is a user-centric performance model. Every web app has these four distinct aspects to its life cycle, and performance fits into them in very different ways:
```
Comment (petele) [inline src/content/en/fundamentals/performance/rail.md]: Small nit - won't block on this, but can we please break lines at 80 characters.

**Comment by petele** [comment]
@PaulKinlan / @kaycebasques  - Please assign to me when this is ready to go.

**Comment by PaulKinlan** [comment]
LGTM.

**Pull request merged**